src/app/config/scripts/prefetch-ssm.js

```diff
@@ -0,0 +1,24 @@
+// Copyright The Linux Foundation and each contributor to CommunityBridge.
+// SPDX-License-Identifier: MIT
+
+const fs = require('fs');
+const RetrieveSSMValues = require('./read-ssm');
+const configVarArray = ['auth0-clientId', 'auth0-domain', 'cla-api-url', 'corp-console-link', 'cla-logo-s3-url'];
```

Comment:
Identify which VD SSM parameters we need.  Candidates are the hard-coded values. These will be different than what we need for EasyCLA.  Notice how the naming convention is:

- parameters typically have a `vd` prefix so that others who view the SSM parameters in AWS will know which product this parameter belongs to. Here, we do not include it as later we add the `vd` prefix and add the {stage} suffix.
- parameters typically have a `{stage}` suffix so that it is obvious that it is for `dev`, `staging` or `prod`.  Here, we do not include the suffix as it is added later when we load the value.

Reply:
Done